Makefile.manual

```diff
@@ -331,6 +331,31 @@ datasets/wikidatawiki.labeled_revisions.w_cache.20k_2015.json: \
 		--host https://wikidata.org \
 		--verbose > $@
 
+datasets/wikidatawiki.sampled_revisions.500k_2018.json:
+	wget -qO- https://quarry.wmflabs.org/run/271461/output/0/json-lines?download=true > $@
+
+datasets/wikidatawiki.autolabeled_revisions.500k_2018.json: \
+		datasets/wikidatawiki.sampled_revisions.500k_2018.json
+	cat $< | \
+	./utility autolabel --host=https://www.wikidata.org \
+		--trusted-groups=bot,sysop,bureaucrat \
+		--trusted-edits=1000 \
+		--revert-radius=3 \
+		--revert-window=48 \
+		--verbose > $@
+
+datasets/wikidatawiki.autolabeled_revisions.500k_2018.no_review.json: \
```

Comment:
We don't need to split these up.  I think we just want to submit the observation that "need_review" to wikilabels.

Reply:
Done